commands/keplr.js

```diff
@@ -234,8 +244,23 @@ const keplr = {
       secretWordsOrPrivateKey,
       password,
       newAccount,
+      walletName,
     );
   },
+
+  async switchWallet({ walletName }) {
+    await module.exports.switchToKeplrIfNotActive();
```

Comment:
why pass the param through an object?

Reply:
its a nicer way to pass params in my opinion since all params are passed as explicit key-value pairs. ordering of params also becomes irrelevant, plus it also allows to send only the params we want instead of having to send every single optional param when we have to send the value of one optional param at the very end.

Reply:
Let's use this where required :)